Simplenote/SPAppDelegate.m

```diff
@@ -599,7 +599,6 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
                 }
             }
         }
-        [_simperium save];
```

Comment:
I think it is fine to remove `save` from here, but check that if we pass content via query string (not sure if anyone uses this) the note is not deleted after we go back to the list.

While we're here, it may be nice to refactor this code to use [URLComponents](https://developer.apple.com/documentation/foundation/urlcomponents) instead of manually parsing the query string.

Reply:
>I think it is fine to remove save from here, but check that if we pass content via query string (not sure if anyone uses this) the note is not deleted after we go back to the list.

Tested and the note persists if there is content in the note.

>While we're here, it may be nice to refactor this code to use URLComponents instead of manually parsing the query string.

refactored the creation to use URL components, much cleaner now.